template.yaml

```diff
@@ -409,17 +416,17 @@ Resources:
             Method: get
             RestApiId: !Ref DataReportApi
 
-  DataLoadingHandler:
+  PersistedResourceCsvTransformer:
     Type: AWS::Serverless::Function
     Properties:
-      Handler: no.sikt.nva.data.report.api.etl.SingleObjectDataLoader::handleRequest
+      Handler: no.sikt.nva.data.report.api.etl.PersistedResourceCsvTransformer::handleRequest
       CodeUri: data-loading
       Policies:
         - !GetAtt S3ManagedPolicy.PolicyArn
         - !GetAtt NeptuneManagedPolicy.PolicyArn
         - !GetAtt SqsManagedPolicy.PolicyArn
         - !GetAtt DeployToVpnPolicy.PolicyArn
-      Timeout: 600
+      Timeout: 30
```

Comment:
Is this change in timeout intentional?

Reply:
Yes. After recent changes in this handler, it is no longer necessary with such a long timeout. 30 seconds is the default and enough.
Also, when a lambda consumes messages from a sqs queue, it can not have a longer timeout than the visibility timeout of the queue (which is default 30 seconds if not set on the queue properties)